src/project/build.scala

```diff
@@ -35,7 +35,7 @@ object Extras {
     // should include these things in the distributed jar
     "io.spray"         %% "spray-json"  % "1.3.2",
     "com.microsoft.CNTK" % "cntk_jni"   % "2.0rc3",
-    "org.opencv"         % "opencv_jni" % "3.2.0"
+    "org.openpnp"         % "opencv" % "3.2.0-1"
```

Comment:
I am getting an error when running tests when using the newest jar, maybe we can do this as part of a separate review as it might involve more changes:
Native code library failed to load.
java.lang.UnsatisfiedLinkError: no Cntk.Core.JavaBinding-2.0d in java.library.path
Exception in thread "Thread-72" java.io.EOFException
        at java.io.ObjectInputStream$BlockDataInputStream.peekByte(ObjectInputStream.java:2917)
        at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1502)
        at java.io.ObjectInputStream.readObject(ObjectInputStream.java:422)
        at org.scalatest.tools.Framework$ScalaTestRunner$Skeleton$1$React.react(Framework.scala:798)
        at org.scalatest.tools.Framework$ScalaTestRunner$Skeleton$1.run(Framework.scala:787)
        at java.lang.Thread.run(Thread.java:748)